tests/Application/src/Filter/AttributeNationalityFilter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Filter;
+
+use App\Grid\Type\NationalityFilterType;
```

Comment:
Please move form type to usual SF placement.

```suggestion
use App\Form\Type\Grid\NationalityFilterType;
```

Reply:
This is not the type of the already existing filter. Last change to this filter is from 2022, I'm not sure this is super useful to make this change in this PR.